app/models/xml/manifest.rb

```diff
@@ -35,12 +43,23 @@ def add_manifest_metadata(manifest_node)
       manifest_node['type'] = @translation.resource.resource_type.name
     end
 
-    def insert_translated_name(manifest_node)
+    def manifest_translated_phrases(manifest_node)
+      phrases = @translation.manifest_translated_phrases
+      # backwards compatibility with manifest name translation:
+      if (i18n_id = title_i18n_id(manifest_node))
+        phrases = phrases.dup
+        phrases[i18n_id] = @translation.translated_name
```

Comment:
`translation.translated_name` could be dropped but I was unsure so I rather kept it, for now

Reply:
does `translation.translated_name` get stored in the database? If it is, I believe it is returned by API calls and is used by the mobile apps.

Reply:
yep, its updated from OneSky (`name_description.xml`'s **name** value) on every translation update